### ISSUE TITLE
inconsistency between the description and YANG model segment for client-svc

### ISSUE THREAD

**Issue by agva123** [haomianzheng/IETF-ACTN-YANG-Model#108]
The description says,

```
A client-svc container is specified to augment the generic TE link
   termination point to describe if the point is capable of carrying
   client signal and what kind of signal can be carried as follow.

        augment /nw:networks/nw:network/nw:node/nt:termination-point
                  /tet:te:
          +--rw client-svc!
             +--rw client-facing?             boolean
             +--rw supported-client-signal*   identityref

   The client-facing is an indicator on whether the point is needed to
   carry client signal.  A list of support-client-signal is used to
   provide the capabilities of client signal specified in
   [I-D.ietf-ccamp-layer1-types].
```

while the code says,
```
     augment /nw:networks/nw:network/nt:link/tet:te
               /tet:te-link-attributes:
       +--rw otn-link!
       |  +--rw odtu-flex-type?   l1-types:odtu-flex-type
       |  +--rw tsg?              identityref
       |  +--rw distance?         uint32
       +--rw client-svc!
          +--rw supported-client-signal*   identityref

```
Should the client-facing be removed because the presence of client-svc indicated that it is client facing?
In this case the description should also be updated.

**Comment by italobusi** [comment]
- [ ] @italobusi : update the YANG model, if needed
- [ ] @haomianzheng : align the text description (and YANG tree extract) with the updated YANG model

**Comment by italobusi** [comment]
> * [x]  @italobusi : update the YANG model, if needed
> *

Fixed in PR #111 

The client-svc is a presence container indicating that the LTP is capable to support CBR clients. In this case, the supported-client-signal shall contain at least one element so a min-element statement has been added to the YANG model

**Comment by italobusi** [comment]
Fixed in [draft-ietf-ccamp-otn-topo-yang-16](https://datatracker.ietf.org/doc/html/draft-ietf-ccamp-otn-topo-yang-16)
